docs/documentation/release-notes.md

```diff
@@ -60,16 +58,16 @@ For single line RNs, follow this format:
 ```markdown
 #### Integrations
 ##### Cortex XDR - IR
-  - Release note here.
+Release note here.
 ```
 
 For single line RNs with a nested list, follow this format:
 ```markdown
 #### Integrations
 ##### Cortex XDR - IR
-  - Release note here.
-    - List item 1
-    - List item 2
+Release note here.
+  - List item 1
+  - List item 2
```

Comment:
@tschanfeld please revert this change to be a nested list (`- Release note here.`)

Reply:
@kgal-pan please check which of the following works with `validate`:
```
- a
  - b
  - c
```
or
```
a 
  -  b
  - c
```

Reply:
@dorschw - Looks like both syntaxes pass validation:

```bash

> file=Packs/SecurityScorecard/ReleaseNotes/1_0_4.md
```

```bash
# Hyphen in root
> cat $file&&demisto-sdk validate -i $file --silence-init-prints

#### Integrations
##### SecurityScorecard
- Release note here
        - Fixed an issue with the *get_last_run* unit test.
        - Updated the Docker image to: *demisto/python3:3.10.4.28442*.
You are using demisto-sdk 1.6.4.

================= Validating file /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md =================

Validating /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md as releasenotes

The files are valid
```
```bash
# No hyphen in root
> cat $file&&demisto-sdk validate -i $file --silence-init-prints

#### Integrations
##### SecurityScorecard
Release note here
        - Fixed an issue with the *get_last_run* unit test.
        - Updated the Docker image to: *demisto/python3:3.10.4.28442*.
You are using demisto-sdk 1.6.4.

================= Validating file /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md =================

Validating /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md as releasenotes

The files are valid
```